infra/base-images/base-runner/coverage

```diff
@@ -82,7 +82,7 @@ function run_fuzz_target {
   local args="-merge=1 -timeout=100 -close_fd_mask=3 $corpus_dummy $corpus_real"
 
   export LLVM_PROFILE_FILE=$profraw_file
-  timeout $TIMEOUT $OUT/$target $args &> $LOGS_DIR/$target.log
+  timeout $TIMEOUT perf record -g -o $DUMPS_DIR/$target.perf $OUT/$target $args &> $LOGS_DIR/$target.log
```

Comment:
I'm not sure we should be doing this in the coverage script. It seems different than coverage IMO.

Reply:
This would surely be useful for timeouts :-)

I also think profiling on the corpus still shows relevant information, as it shows profiling across all kinds of inputs...

I did it in the coverage script, as it was easy : get both reports in one shot...

Let me know what you prefer

Reply:
Profiling on the corpus seems like it will produce a bit of a skewed result because the corpus is a subset of the inputs that the target runs on during fuzzing. Let's say there is an edge that is hit very frequently during fuzzing and takes a really long time, but only one file in the corpus covers it. The profiling won't show that fuzzing is spending a ton of time on this edge. 
@oliverchang What do you think about this?

Reply:
Another reason this is problematic: In this run the target is instrumented with Source-based coverage instrumentation and dumping results. So not only will profiling include time wasted on that stuff, it will not be able to accurately show the amount of time spent on sancov instrumentation which can also be useful to know.

Reply:
Indeed, so what is the best way to run it ?
We can run it for some definite time with classic ASAN instrumentation
But then, should it be a new subcommand to `infra/helper.py` ? Or an option to the `run_fuzzer` subcommand ?

Reply:
How useful is sancov info? It seems like pretty niche information in most cases? 

That said, it does seem like something that could be built more into ClusterFuzz instead, as we may want this data for timeouts and there's the potential corpus vs actual fuzzing skew issue. Not requiring a specialised build also make it easier. 

I think this is fine to include with coverage as is for now though, so we can evaluate this before devoting more time to properly integrate this into ClusterFuzz. How long do these runs typically take?